### ISSUE TITLE
Signing in from audit page does not update UI with older contributions

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#1298]
When you sign in from the audit page, the UI acts as if you had just signed up, in that the number of labels contributed in that neighborhood is not updated, if the page initially loaded with the tutorial it is not refreshed to skip the tutorial, and the messages shown to first time users will be shown (the last issue is captured in #788 ).

I think the easiest way to deal with this is to just refresh the page if you sign in from the audit page. We specifically moved away from doing this in PR #786, but that was meant to make it not look like their mission was reset when an anon user signs _up_. I would think that this is less of a worry when a user signs _in_, and we could safely refresh the audit page without causing confusion in almost every case.

And I would either like to implement this or #783 (assigning anon user contributions to registered user account after they sign in), otherwise we run into a weird situation of half a mission being completed as an anon user and half as a registered user, where half of the contributions are given to different accounts.

**Comment by misaugstad** [comment]
To make it more clear, I am suggesting just refreshing the page to solve this issue :)

**Comment by jonfroehlich** [comment]
Whoever works on this needs to come up with a clear, repeatable way to test.

**Comment by DoctorWarrior** [comment]
I think I will write a design spec first before I actually start modifying the code base.

**Comment by DoctorWarrior** [comment]
Hello, I just applied for an account and did several tests. There are several situations I noticed that are different from what you described:
1. When I signed in, the tutorial did not show up. The welcome message "your first mission is to..." showed up. I think this is ok since I would like to have that reminder if I am in the middle of a mission.
2. There is a potential tag-count bug: at first, I had 2 tags. However, after refreshing the page, the number of tags contributed in that neighborhood showed only 1. I tried it a second time and entered 7 tags. After refreshing I had only 3 left. A third time I entered 8 tags, but only had 6 left after refreshing.
I left plenty of time before refreshing (3 minutes) for the database to update but the counter was still not correct. In addition, after refreshing, all tags on the map disappeared, even though the "number of tags" counter showed numbers > 0.

Can you try again and see if you can reproduce these? Do you know if there have been any changes in the code since you reported your issues? Let's get a consistent list of problems and start troubleshooting them.

**Comment by misaugstad** [comment]
>When I signed in, the tutorial did not show up.

I meant that if the page loads and you are already in the tutorial, then you sign in, the tutorial _stays up_. I didn't mean that when you sign in the tutorial appears if it wasn't already there.

>There is a potential tag-count bug:

I'm assuming you mean label-count here.

>I had 2 tags. However, after refreshing the page, the number of tags contributed in that neighborhood showed only 1

This is something we've decided not to fix right now. The labels you place before signing in aren't being counted towards your label count after sign-in + page refresh. We figured that people rarely audit as an anonymous user for an extended period of time before signing in to an account that they already have. Typically this would be an accident, and one would realize during their first mission when they are invited to sign up that they must not be logged in. Thus we don't think it is important to attribute that contribution as anon user to their registered account.

We really just need to refresh the page so once someone realizes they aren't logged in and choose to log in, they are put back where they were last time they logged in. And we ignore the small contribution they just made anonymously.
